src/mbgl/text/cross_tile_symbol_index.cpp

```diff
@@ -86,11 +96,29 @@ void CrossTileSymbolLayerIndex::handleWrapJump(float newLng) {
     lng = newLng;
 }
 
-bool CrossTileSymbolLayerIndex::addBucket(const OverscaledTileID& tileID, SymbolBucket& bucket, uint32_t& maxCrossTileID) {
-    const auto& thisZoomIndexes = indexes[tileID.overscaledZ];
+namespace {
+
+bool isInVewport(const mat4& posMatrix, const Point<float>& point) {
```

Comment:
@pozdnyakov should this be spelled `isInViewport`?

Reply:
yeah :grimacing: thanks for the catch, will fix it in a following patch..